lib/services/bases/chat_bot.dart

```diff
@@ -0,0 +1,13 @@
+import '../../models/ApplicationFunction.dart';
+import '../../utility/RouteGenerator.dart';
+
+/**
+ * Abstract implementation for ChatBot Service functions
+ * to be publicly accessible
+ */
```

Comment:
Canonical dart doesn't use block comments:
https://dart.dev/guides/language/effective-dart/documentation#dont-use-block-comments-for-documentation

*Same comment applies for all files*